modules/auxiliary/admin/ubiquiti/ubiquiti_config.rb

```diff
@@ -0,0 +1,94 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'zlib'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Ubiquiti
+
+  def initialize(info={})
+    super( update_info( info,
+      'Name'          => 'Ubiquiti Configuration Importer',
+      'Description'   => %q{
+        This module imports an Ubiquiti device configuration.
+        The db file within the .unf backup is the data file for
+        Unifi. This module can take either the db file or .unf.
+        },
+      'License'       => MSF_LICENSE,
+      'Author'        => [ 'h00die'],
```

Comment:
This is my contribution.

```suggestion
      'Author'        => ['h00die'],
```